pywr-schema/src/error.rs

```diff
@@ -52,8 +53,18 @@ pub enum SchemaError {
     #[error("Metric F64 error: {0}")]
     #[cfg(feature = "core")]
     CoreMetricF64Error(#[from] pywr_core::metric::MetricF64Error),
-    #[error("Error loading data from table `{0}` (column: `{1:?}`, index: `{2:?}`) error: {error}", table_ref.table, table_ref.column, table_ref.index)]
-    TableRefLoad { table_ref: TableDataRef, error: TableError },
+    #[error("Error loading data from table `{0}` (column: `{1:?}`, index: `{2:?}`) error: {source}", table_ref.table, table_ref.column, table_ref.index)]
+    TableRefLoad {
+        table_ref: TableDataRef,
+        #[source]
+        source: Box<TableError>,
+    },
+    #[error("Circular node reference(s) found.")]
+    CircularNodeReference,
```

Comment:
Looks like these have been added back in when merging main. These are now network build errors.

Reply:
Good spot. I'll fix that.